crates/chain/src/genesis.rs

```diff
@@ -0,0 +1,54 @@
+//! Bitcoin genesis hashes.
```

Comment:
Why can't we use `rust-bitcoin` for this?

https://docs.rs/bitcoin/latest/bitcoin/blockdata/constants/fn.genesis_block.html#

Reply:
I didn't know this existed. 😮

Reply:
Removed.